files/build_templates/docker_image_ctl.j2

```diff
@@ -327,6 +327,11 @@ function postStartAction()
             docker cp $PSENSOR pmon:/usr/bin/
         fi
     fi
+{%- elif docker_container_name == "eventd" %}
+    export EVENTD_STATE=$(sonic-db-cli -s CONFIG_DB HGET 'FEATURE|eventd' 'state')
+    j2 -f json --import-env=ENVIRONMENT /usr/share/sonic/templates/rsyslog_plugin/rsyslog_plugin.conf.j2 /usr/share/sonic/templates/rsyslog_plugin/events_info.json  > /etc/rsyslog.d/host_events.conf
+    j2 -f json --import-env=ENVIRONMENT /usr/share/sonic/templates/rsyslog_plugin/rsyslog_plugin.conf.j2 /usr/share/sonic/templates/rsyslog_plugin/syncd_events_info.json  > /etc/rsyslog.d/syncd_events.conf
```

Comment:
@zbud-msft @qiluo-msft These commands are CPU heavy as well as those added in docker-init.j2 for multiple containers.
Any performance measurements done w.r.t fast-reboot, warm-reboot to understand the impact?
Any idea how to achieve same functionality without introducing additional CPU overhead during init?